internal/pkg/cli/env_init.go

```diff
@@ -61,6 +61,19 @@ type InitEnvOpts struct {
 	*GlobalOpts
 }
 
+// Validate returns an error if the values passed by the user are invalid.
+func (opts *InitEnvOpts) Validate() error {
+	if opts.EnvName != "" {
+		if err := validateEnvironmentName(opts.EnvName); err != nil {
+			return err
+		}
+	}
+	if opts.ProjectName() == "" {
+		return errors.New("no project found, run `project init` first please")
```

Comment:
nit: this can also result from the fact that they are not in a workspace. So maybe we can reword the error massage like "no project found, run `project init` to create a project or `cd` to a workspace" (or something with similar meaning).

Reply:
Done!